src/index.tsx

```diff
@@ -1,4 +1,4 @@
-import Dropdown from './dropdown';
+import Dropdown from './DropDown';
```

Comment:
I have fixed the capitalisation issue of this file name, it will be dropdown.tsx